osu.Game/Screens/OsuScreen.cs

```diff
@@ -128,5 +147,33 @@ protected override bool OnExiting(Screen next)
             Beatmap.UnbindAll();
             return false;
         }
+
+        private void logoSetup(bool resuming) => LogoSetup(logo, resuming);
+
+        protected virtual void LogoSetup(OsuLogo logo, bool resuming)

```

Comment:
We are not prefixing methods like this in other parts of the game. Better is `SetupLogo`.